message-switch-unix.opam

```diff
@@ -1,30 +1,35 @@
 # This file is generated by dune, edit dune-project instead
-license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
 opam-version: "2.0"
-name: "message-switch-unix"
-maintainer: "xen-api@lists.xen.org"
-authors: [ "xen-api@lists.xen.org" ]
-homepage: "https://github.com/xapi-project/xen-api"
+synopsis: "A simple store-and-forward message switch"
+description:
+  "The switch stores messages in queues with well-known names. Clients use a simple HTTP protocol to enqueue and dequeue messages."
+maintainer: ["Xapi project maintainers"]
+authors: ["xen-api@lists.xen.org"]
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+homepage: "https://xapi-project.github.io/"
 bug-reports: "https://github.com/xapi-project/xen-api/issues"
-dev-repo: "git+https://github.com/xapi-project/xen-api.git"
-tags: [ "org:xapi-project" ]
-build: [
-  ["./configure" "--prefix" "%{prefix}%"]
-  [ "dune" "build" "-p" name "-j" jobs ]
-]
 depends: [
-  "ocaml"
```

Comment:
no longer depends on OCaml?

Reply:
Unless we want to specify a version, the line is useless as ocaml will be installed because of dune